webapp/components/create_comment.jsx

```diff
@@ -66,11 +69,33 @@ export default class CreateComment extends React.Component {
 
     componentDidMount() {
         PreferenceStore.addChangeListener(this.onPreferenceChange);
+
         this.focusTextbox();
+        document.addEventListener('keydown', this.showShortcuts);
     }
 
     componentWillUnmount() {
         PreferenceStore.removeChangeListener(this.onPreferenceChange);
+        document.removeEventListener('keydown', this.showShortcuts);
+    }
+
+    showShortcuts(e) {
```

Comment:
`/shortcuts` not be called twice, because `keydown` event is overwritten. 
But this function is unnecessary in here. I'll remove this.